### PR TITLE
Start cleaning up

### PR DESCRIPTION
**Pull request by tekknolagi**
- [x] Remove vendored `topkg`, replace with opam package.
- [x] Remove Docker stuff
- [x] Remove `upgradeSyntax.sh`
- [x] Remove unused `CHANGELOG.md`

### CODE REVIEW

**Comment by yunxing** [comment]
Thanks. CI is not happy. Could you take a look?

**Comment by tekknolagi** [comment]
Yep, I'll mark WIP.

**Comment by tekknolagi** [comment]
IT'S ALIVE (well, we'll see after CI)

File: pkg/topkg_old.ml [hunk 1]
```diff
@@ -275,6 +277,7 @@ end = struct
 
   let native = bool "native"
   let native_dynlink = bool "native-dynlink"
+  let utop = bool "utop"
   let error () = !error
 end
```
Comment (yunxing) [inline pkg/topkg_old.ml]: Should we remove this file?

Comment (tekknolagi) [inline pkg/topkg_old.ml]: Yep

**Comment by yunxing** [comment]
Thanks for working on this! Let's change '_build/pkg/build.native' to 'build.native' then we should be good to merge.

**Pull request merged**